### PR TITLE
Implemented get supported asset

### PR DESCRIPTION
**Pull request by azeezlala**

### CODE REVIEW

**Review by odetolakehinde — APPROVED** [review]

File: api/api.go [hunk 1]
```diff
@@ -41,6 +41,7 @@ type RemoteCalls interface {
 	// Wallet APIs
 	GetWallet(ctx context.Context, request model.WalletRequest) (model.Wallet, error)
 	GetWallets(ctx context.Context, customerID string) ([]*model.Wallet, error)
+	SupportedAsset(ctx context.Context) ([]*model.SupportedCurrencies, error)
```
Comment (odetolakehinde) [inline api/api.go:44]: `GetSupportedAssets`

File: model/model.go [hunk 2]
```diff
@@ -310,4 +310,10 @@ type (
 		CreatedAt     time.Time  `json:"created_at"`
 		UpdatedAt     *time.Time `json:"updated_at"`
 	}
+
+	// SupportedCurrencies data object
+	SupportedCurrencies struct {
```
Comment (wharleyinc) [inline model/model.go:315]: `SupportAssets` since we are dealing with asset

Comment (wharleyinc) [reply model/model.go]: Apologies @azeezlala .

I mean `SupportAsset`

Comment (wharleyinc) [reply model/model.go]: `SupportedAsset`

File: main.go [hunk 3]
```diff
@@ -130,5 +130,11 @@ func main() {
 		fmt.Printf("Error: %v\n", err)
 		return
 	}
-	fmt.Printf("all wallet info: %+v\n", *allWallet[0])*/
+	fmt.Printf("all wallet info: %+v\n", *allWallet[0])
+	assets, err := apiCalls.SupportedAsset(ctx)
```
Comment (wharleyinc) [inline main.go:134]: Here

**Review by odetolakehinde — APPROVED** [review]

**Review by wharleyinc — APPROVED** [review]

**Pull request merged**